spark/src/test/scala/org/apache/comet/CometExpressionCoverageSuite.scala

```diff
@@ -123,7 +134,7 @@ class CometExpressionCoverageSuite extends CometTestBase with AdaptiveSparkPlanH
             Seq(("", "No examples found in spark.sessionState.functionRegistry"))))
     }
 
-    // TODO: convert results into HTML
+    // TODO: convert results into HTML or .md file
```

Comment:
Perhaps log an issue for this and refer to that instead? 
It would be great to link the output directly into the documentation.

Reply:
Yes, in next PR its expected to generate MD file in docs, rather than txt

Reply:
Can you remove the commented out code from this PR?